.changeset/three-hats-attack.md

```diff
@@ -0,0 +1,11 @@
+---
+"@saleor/react-hook-form-macaw": patch
+"saleor-app-emails-and-messages": patch
+"saleor-app-products-feed": patch
+"saleor-app-invoices": patch
+"saleor-app-search": patch
+"@saleor/apps-ui": patch
+"saleor-app-crm": patch
+---
+
+Update the Macaw UI library to the latest release
```

Comment:
for the future
- try to include version (pre-95) - it can be helpful
- contrary to commits, changelog should be in past simple, "updated macaw"
- this change CAN have unnecessary side effects so it also should be included here

Better message can be:


>Updated @saleor/macaw-ui to 0.8.0-pre.95. This version introduces change in spacing scale, so there may be slight changes in spacing